### ISSUE TITLE
ModuleNotFoundError: No module named 'readit' error when using Python 3

### ISSUE THREAD

**Issue by bhavin192** [pythonpune/readit#51]
```shell
Traceback (most recent call last):
  File "/home/bhavin192/readit/py36/bin/readit", line 11, in <module>
    load_entry_point('readit==0.1', 'console_scripts', 'readit')()
  File "/home/bhavin192/readit/py36/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 572, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/bhavin192/readit/py36/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 2749, in load_entry_point
    return ep.load()
  File "/home/bhavin192/readit/py36/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 2402, in load
    return self.resolve()
  File "/home/bhavin192/readit/py36/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 2408, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/bhavin192/readit/py36/lib64/python3.6/site-packages/readit-0.1-py3.6.egg/manager/__init__.py", line 1, in <module>
    from readit import main
ModuleNotFoundError: No module named 'readit'
```

#### Steps to reproduce:
- Create virtual env using `python3 -m venv py36`
- `$ python setup.py install`
- `$ readit`

#### Details:
Python version: `Python 3.6.4`

**Comment by shitalmule04** [comment]
I think this error because of [Circular Imports](http://stackabuse.com/python-circular-imports/). While writing using pytest+click.testing, the similar error occurred. 

```
Hint: make sure your test modules/packages have valid Python names.
Traceback:
shital/readit/tests/test_manager.py:3: in <module>
    from readit import readit as rd
E   ImportError: No module named readit

```

**Comment by ganeshhubale** [comment]
@bhavin192 
Will it be ok, if we make **readit** compatible with Python3 only. because now we are going to import modules according to Python3 to fix #51 which is not supported in Python2. also please click here to know about [Drop Python 2 support by 2020](http://www.asmeurer.com/blog/posts/moving-away-from-python-2/).

**Comment by bhavin192** [comment]
@ganeshhubale it's upto you, some of the packages are maintained for both Python 2 and 3. So it's not impossible to do that, but I agree it becomes a bit hard to maintain compatibility between both 2 and 3.
My personal opinion would be to drop support for Python 2.
